### ISSUE TITLE
Trying to report a test not in a fiber! You probably forgot to wrap a callback in bindEnvironment.

### ISSUE THREAD

**Issue by dandv** [meteor/meteor#3673]
```sh
git clone git@github.com:MeteorPackaging/pdftk.git
meteor test-packages pdftk/
```

Running the async test, which calles Node's `execFile`, yields:

```
 Trying to report a test not in a fiber! You probably forgot to wrap a callback in bindEnvironment.
 (STDERR) Trace
 (STDERR)     at [object Object].onReport (packages/tinytest/tinytest_server.js:46:1)
 (STDERR)     at [object Object]._.extend._report (packages/tinytest/tinytest.js:574:1)
 (STDERR)     at test.run.totalTime (packages/tinytest/tinytest.js:454:1)
 (STDERR)     at [object Object].wrappedOnEvent [as onEvent] (packages/tinytest/tinytest.js:361:1)
 (STDERR)     at [object Object]._.extend.fail (packages/tinytest/tinytest.js:85:1)
 (STDERR)     at packages/local-test:pdftk:pdftk/pdftk_pdftk-tests.js:5:1
 (STDERR)     at packages/pdftk:pdftk/pdftk-wrapper.js:17:1
 (STDERR)     at ChildProcess.exithandler (child_process.js:652:7)
 (STDERR)     at ChildProcess.emit (events.js:98:17)
 (STDERR)     at maybeClose (child_process.js:756:16)
```

**Comment by glasser** [comment]
I mean, it sounds like you didn't wrap a callback in bindEnvironment.

**Comment by dandv** [comment]
Solved the problem by using [wrapAsync](https://github.com/MeteorPackaging/pdftk/blob/master/pdftk-wrapper.js#L11) around the execFile call.

Would it be better to *only* wrap [the callback in the async test](https://github.com/MeteorPackaging/pdftk/blob/master/tests/pdftk_pdftk-tests.js#L7) in `Meteor.bindEnvironment`, as the error message suggests, though?

I tried that and everything worked, but it seems that `wrapAsync` at the package level is the more solid solution than just `bindEnvironment` in the test. However, the package itself functions correctly without `wrapAsync` around the `execFile` call.
